AuthorYYMMDD_fignr_001.yaml

```diff
@@ -0,0 +1,74 @@
+# This file can exist with the "source" information only, the entry data exists should be marked as "False" 
+# to indcate that neither the metadata nor the actual data were digitized or provided.
+# use "null" to idicate that a value is not available in the literature
+version: 1 # Is this necessary. Isn't this anyway given by git versioning

```

Comment:
It's not given by git versioning. We should have this so we can mix different versions (when we move from one to the other.)

Reply:
```suggestion
version: 1
```